### ISSUE TITLE
Feature request: simple timeout 

### ISSUE THREAD

**Issue by cocoonkid** [will-stone/browserosaurus#244]
It would be great if I could enable a simple timeout with a small slider during which browsersaurus would open the default set browser as usual without confirmation.

This is immensely helpful during scheduled email or im message cleanup and sorting.

Yes, I currently simply disable it for the duration bit often i forget to reactivate it.

Yes, i know this is probably only helpful for real power users but that would be worth at least a coffee to me haha.

**Comment by will-stone** [comment]
Thanks for your offer of money to implement a feature, however I (and most open source maintainers) will not add features for money. If you are a fan of Browserosaurus, the most flattering thing you can do is recommend it to your friends and family, and consider [sponsorship](https://github.com/sponsors/will-stone). I highly recommend [this article](https://medium.com/@teejeetech/etiquette-in-open-source-projects-f302b6e19c38) by Tony George which summarises this well in point 10.

I have an ethos for Browserosaurus that I really should formalise on the website some time. Essentially, I would like Browserosaurus to be predictable. In programming language terms this is often described as _pure_; given the same input, the output will always be the same. I feel the muscle-memory of clicking a link and knowing you need to hit space to open your favourite browser **every time** is far more powerful than tweaks that temporarily shave a few half-seconds from your workflow. I received a lot of requests for a rules engine, which I always turned-down. Despite this being something I am unlikely to use, I understand that others would appreciate this functionality. This is why I have teamed-up with John Sterling and highly recommend users check out his open source software [Finicky](https://github.com/johnste/finicky). This can be used in tandem with Browserosaurus to have the best of both worlds. I know this doesn't quite resolve your need to temporarily disable Browserosaurus but maybe it will help with understanding why it works this way, and maybe you could use Finicky to create a rule for your email and IM apps.

**Comment by cocoonkid** [comment]
Thank you for the thorough explanation and noting Finicky.
I did not intend to bribe you into adding this feature.

**Comment by will-stone** [comment]
No worries man, it's cool. I appreciate all the ideas you are coming-up with 😁

When I started-out in open source, as a consumer, I made lots of classic mistakes. Most notably things like: "why isn't this fixed yet?!!" or the dreaded "+1" comment. It's only really since releasing my own software and packages that I've come to appreciate the etiquette that others have outlined in articles and blog posts. I almost feel that releasing at least one package, big or small, should be the right of passage into open source 🤔 Anyway, daydreaming over... 😆
